### PR TITLE
Add helper for repository tables in docx report [SCI-4157]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-4157](https://biosistemika.atlassian.net/browse/SCI-4157)

### What was done
Add helper for repository tables in docx report

### CODE REVIEW

File: app/services/reports/docx/repository_helper.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    repository_data[:data].repository_rows.each do |record|
+      row = []
+      row.push(record.id)
+      row.push(escape_input(record.name))
+      row.push(I18n.l(record.created_at, format: :full))
+      row.push(escape_input(record.created_by.full_name))
+
+      custom_cells = record.repository_cells
+      repository_data[:data].mappings.each do |column_id, _position|
+        value = custom_cells.find_by(repository_column_id: column_id)&.value&.formatted
```
Comment (okriuchykhin) [inline app/services/reports/docx/repository_helper.rb]: So it loads each cell and value separately?

Comment (aignatov-bio) [reply app/services/reports/docx/repository_helper.rb]: I need check for empty cells and place them in correct order. In `mappings` we have correct column order to match with header.

Comment (okriuchykhin) [reply app/services/reports/docx/repository_helper.rb]: It is not right way to do it as it will generate tons of extra SQL requests

Comment (aignatov-bio) [reply app/services/reports/docx/repository_helper.rb]: Changed

**Pull request merged**